packages/manager/apps/web/client/app/hosting/multisite/hosting-multisite.controller.js

```diff
@@ -129,6 +132,75 @@ angular
               }
             });
           })
+          .then(async () => {
+            $scope.user = await WucUser.getUser();
+          })
+          .then(() => {
+            const getNichandleAttached = async (
+              diagnosticSuffix,
+              domain,
+              nichandle,
+            ) => {
+              const serviceInfos = await HostingDomain.getServiceInfosForZone(
+                domain.name.replace('www.', ''),
+              );
+              const { contactTech, contactAdmin } = serviceInfos;
+
+              let isDnsAttachedToNic = false;
+              if (nichandle === contactAdmin || nichandle === contactTech) {
+                isDnsAttachedToNic = true;
+              }
+              return {
+                diagnosticState: diagnosticSuffix[domain.name],
+                isDnsAttachedToNic,
+              };
```

Comment:
```suggestion
              const { contactTech, contactAdmin } = serviceInfos;
              const isDnsAttachedToNic = [contactTech, contactAdmin].includes(nichandle);
              return {
                diagnosticState: diagnosticSuffix[domain.name],
                isDnsAttachedToNic,
              };
```

Reply:
this one has been previously simplified this way

![image](https://github.com/ovh/manager/assets/8767496/51db2d90-6689-48f2-9577-fd866800c2a7)